tests/test_views.py

```diff
@@ -631,3 +631,64 @@ def test_when_correct_otp_email_passed(self):
         response = self.client.post(self.url, data=self.data_correct_otp, format="json")
 
         self.assertEqual(202, response.status_code)
+
+
+class TestUploadImageView(APITestCase):
+    """UploadImageView Test"""
+
+    def setUp(self) -> None:
+        """SetUp test data"""
+        self.url = reverse("upload_profile_image")
+
+        self.user = baker.make(
+            "drf_user.User",
+            username="user",
+            email="user@email.com",
+            name="user",
+            mobile=1234569877,
+            is_active=True,
+        )
+
+        self.auth_transaction = baker.make(
+            "drf_user.AuthTransaction",
+            created_by=self.user,
+        )
+
+        # create otp of registered user
+        self.user_otp = baker.make(
+            "drf_user.OTPValidation", destination="user@email.com", otp=437474
+        )
```

Comment:
@iamvivekkaushik why are we creating this data, since we're using `force_authenticate`